readme.md

```diff
@@ -42,10 +44,18 @@ bind -n M-b run 'harpoon -a'
 bind -n .   run 'harpoon -A'
 bind -n M-v run 'harpoon -l'
 bind -n M-i run 'harpoon -e'
-bind -n M-n run 'harpoon -s 1'
-bind -n M-e run 'harpoon -s 2'
-bind -n M-o run 'harpoon -s 3'
-bind -n M-s run 'harpoon -s 4'
+bind -n M-q run 'harpoon -s 1'
+bind -n M-w run 'harpoon -s 2'
+bind -n M-e run 'harpoon -s 3'
+bind -n M-r run 'harpoon -s 4'
+
+# Map panes or sessions to specific by replacing entires
+# If there is no pane/session in the given index, it will be appended to the list instead.
+bind -n M-q run 'harpoon -s 1'  # alt+q goes to index 1
```

Comment:
this is repetition of the command above, either remove this entry or the one above. I would suggest the following

```sh
bind -n M-i run 'harpoon -e'
bind -n M-q run 'harpoon -s 1' # alt+q goes to index 1
bind M-q run 'harpoon -r 1'     # prefix alt+q replace entry index 1 with current session
bind -n M-w run 'harpoon -s 2'
bind M-w run 'harpoon -R 2'     # replace entry at index 2 with current pane within session
bind -n M-e run 'harpoon -s 3'
bind -n M-r run 'harpoon -s 4'

# Note: If there is no entry at the given index, it is appended to the list instead.
```

Reply:
Good suggestion!